lib/fathead/python_six/parse.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+import glob
+import sys
+
+python_six_base_url = 'https://pythonhosted.org/six/'
+
+class SixModule(object):
+    '''
+    Class for storing all necessary information about the python six lib. This
+    information is then used to create the output.txt file using the
+    __str__() method on this class
+    '''
+    def __init__(self, name, description, filename):
+        
+        self.name = name
+        self.description = description
+        self.filename = filename
+        self.usage = ''
+
+    def __str__(self):
+        '''
+        Output the python six information in the proper format required for
+        DuckDuckGo Fatheads
+        '''
+
+        # Clean up the usage statement which can have newline characters and
+        # tab characters, which mess up how it renders 
+
+        code =  '<pre><code>{}</code></pre>'.format(self.usage.replace('\n','\\\\n'))
```

Comment:
`.replace('\n','\\\\n'))` should be `.replace('\n','\\n'))` -- we're replacing the `\n` with an escaped newline `\\n` which becomes the literal string `\n` in the output.txt

a `\\n` in the output.txt will render the literal string `\n` on the page which we don't want